edb/edgeql/ast.py

```diff
@@ -433,14 +433,18 @@ class ReturningStatement(Statement):
     result_alias: str
 
 
-class SelectQuery(ReturningStatement):
+class AllFiltersStatement(Statement):
```

Comment:
I'd prefer this to be called `SelectClauseStatement` or something like that.

Reply:
OK